frontend/src/components/Pricing.tsx

```
@@ -127,15 +126,17 @@ const Pricing = () => {
                   </p>
                   <div className="flex flex-col text-gray-300 items-center my-8">
-                    <p className="font-extrabold text-4xl">{starterPlan.price}</p>
+                    <p className="font-extrabold text-4xl">
+                      {starterPlan.price}
+                    </p>
                     {/* <p className="text-sm opacity-60">/month</p> */}
                   </div>
                 </div>
                 <div className="flex flex-col gap-1">
                   {starterPlan.features.map((feature) => (
                     <p className="flex items-center gap-2 text-sm text-gray-300">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)